tests/integration/list/resources/resources_integ_base.py

```diff
@@ -0,0 +1,32 @@
+import os
+from unittest import TestCase
+from pathlib import Path
+import uuid
+import shutil
+import tempfile
+from tests.integration.list.list_integ_base import ListIntegBase
+
+
+class ResourcesIntegBase(ListIntegBase, TestCase):
```

Comment:
`ListIntegBase` is extending from `TestCase`  not need to extend here as well.

Reply:
Removed, thanks